lib/registries/addon/overview/controller.ts

```diff
@@ -2,22 +2,65 @@ import { action, computed } from '@ember-decorators/object';
 import { alias, not } from '@ember-decorators/object/computed';
 import { service } from '@ember-decorators/service';
 import Controller from '@ember/controller';
+import config from 'ember-get-config';
+import I18N from 'ember-i18n/services/i18n';
 import Media from 'ember-responsive';
 
 import Registration from 'ember-osf-web/models/registration';
 import { GuidRouteModel } from 'ember-osf-web/resolve-guid/guid-route';
+import CurrentUser from 'ember-osf-web/services/current-user';
+import pathJoin from 'ember-osf-web/utils/path-join';
+
+enum RegistrationState {
+    EMBARGOED = 'embargoed',
```

Comment:
We've not been completely consistent, but most of our enums use PascalCase for the names, and I think it's a little more readable:
```ts
enum RegistrationState {
  Embargoed = 'embargoed',
  Public = 'public',
  // ...
}
```